src/metric/populate_test.go

```diff
@@ -147,13 +147,18 @@ func TestPopulateK8s(t *testing.T) {
 	assert.Error(t, err)
 
 	// Expected errs (missing data)
+	// TODO not good to compare error strings...
 	expectedErrs := []error{
 		errors.New("error populating metric for entity ID kube-system_newrelic-infra-rz225: cannot fetch value for metric deploymentName, metric not found"),
 		errors.New("error populating metric for entity ID kube-system_newrelic-infra-rz225: cannot fetch value for metric reason, metric not found"),
 		errors.New("error populating metric for entity ID kube-system_newrelic-infra-rz225: cannot fetch value for metric message, metric not found"),
 		errors.New("error populating metric for entity ID kube-system_newrelic-infra-rz225_newrelic-infra: cannot fetch value for metric deploymentName, metric not found"),
 		errors.New("error populating metric for entity ID kube-system_newrelic-infra-rz225_newrelic-infra: cannot fetch value for metric cpuLimitCores, metric not found"),
 		errors.New("error populating metric for entity ID kube-system_newrelic-infra-rz225_newrelic-infra: cannot fetch value for metric reason, metric not found"),
+		errors.New("error populating metric for entity ID kube-system_newrelic-infra-rz225_newrelic-infra: cannot fetch value for metric cpuCoresUtilization, 'cpuUsedCores' is nil"),
```

Comment:
Should we at least create a constant for these expected values?

Reply:
I guess it is not ideal since they are kind of generated 😕

Reply:
yes, checking strings is not the best option. We should create some kind of error type for these. I just didn't have time in this pr

Reply:
Let's do a release a first and then  I'll do another PR to change this. It's not a critical issue at the moment